examples/proxy/main.tf

```diff
@@ -1,3 +1,16 @@
+resource "random_pet" "main" {
+  length = 1
+}
+
+module "secrets" {
```

Comment:
I can't comment on that line since it wasn't changed, but line 84 should call this output:
```
  license_secret           = module.secrets.license_secret
```